[DOCS] Adding Conan distribution article to docs for master (#20205)
openvinotoolkit/openvino: docs/articles_en/get started/installing-openvino-overview/installing-openvino-shared/installing-openvino-conan.md

# Install OpenVINO™ Runtime from Conan Package Manager {#openvino_docs_install_guides_installing_openvino_conan}

@sphinxdirective

.. meta::
   :description: Learn how to install OpenVINO™ Runtime on Windows, Linux, and 
                 macOS operating systems, using Conan Package Manager.

.. note::
   

>    Note that the Conan Package Manage distribution:

Reviewer: ```suggestion
   Note that the Conan Package Manager distribution:
```

>    * is dedicated to users of all OSs (Windows, Linux, macOS, Solaris, FreeBSD, etc.)

Reviewer: ```suggestion
   * is dedicated to users of all major OSs: Windows, Linux, macOS
```

>    * it can be used for any existing platfrom: desktop, mobile, embbeded, servers, bare metal, etc.

Reviewer: ```suggestion

```

>    * it can be used for any existing platfrom: desktop, mobile, embbeded, servers, bare metal, etc.
>    * can manage any compiler in any of their versions.
>    * OpenVINO™ is included in the Conan Center Index - registry with the software packages.

Reviewer: ```suggestion
  
```
